services/app/src/components/nameRoute/NameRoute.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
+import { Outlet } from 'react-router-dom';
+import WindowPrompt from '../prompt/WindowPrompt';
+
+const NameRoute = () => {
+  const [user, setUser] = useState('');
+  const [auth, setAuth] = useState(false);
+
+  useEffect(() => {
```

Comment:
The entire user & auth management side looks like it could be a custom hook that returns user and auth states. That way in `services/app/src/pages/game/Game.jsx:15` you wouldn't need to know what the key to the local storage of username is and you'd just use the hook.

Reply:
should be done in a correct manner, ty for noticing :100: